packages/react/src/components/Popover/index.tsx

```diff
@@ -98,6 +98,11 @@ interface PopoverBaseProps {
    */
   align?: PopoverAlignment;
 
+  /**
+   * Will add padding value to arrow in case of bottom-left, bottom-right or top-left, top-right
+   */
+  arrowPadding?: number;
```

Comment:
```suggestion
  caretPadding?: number;
```

This is minor, but the website docs exclusively refer to it as a caret and not an arrow.

Reply:
Sounds good to me, changes applied as suggested.